JobWorkers/DemoExperiments/DemoManualRequest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace JobWorkers\DemoExperiments;
+
+class DemoManualRequest extends \JobWorkers\JobWorkerManualRespond {
+  /** @var int */
+  public $num_to_x2;
+
+  public function __construct(int $num_to_x2) {
+    $this->num_to_x2 = $num_to_x2;
+  }
+
+
+  function handleRequest(): void {
+    $response = new DemoArrX2Response();    // respond with this class also, not to create a duplicate, not the point
+    $response->arr_x2 = [$this->num_to_x2 ** 2];
+
+    $this->respondAndContinueExecution($response);
+
+    sleep(1);
+    // log("DemoManualRequest finish", $this->num_to_x2);
```

Comment:
Do we intend to keep these commented-out log calls?

Reply:
Dropped